class RetryError(Exception):
    """Exception raised when retry"""
    pass
